src/PowerShellEditorServices/Services/TextDocument/SemanticToken.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+
+namespace Microsoft.PowerShell.EditorServices.Services.TextDocument
+{
+    class SemanticToken
```

Comment:
Isn't there an Omnisharp type for this?

Reply:
No, never made one because it adds extra allocations.

Reply:
I can add one if you'd like

Reply:
I do have `NormalizedToken` that I used for unit testing purposes.  Emits everything as an easy to compare string.

https://github.com/OmniSharp/csharp-language-server-protocol/blob/master/test/Lsp.Tests/SemanticTokensDocumentTests.cs#L280-L353